docs/content/components/avatar.md

```diff
@@ -2,7 +2,7 @@
 title: Avatar
 status: Beta
 source: https://github.com/primer/view_components/tree/main/app/components/primer/avatar_component.rb
-storybook: https://primer-view-components.herokuapp.com/?path=/story/primer-avatar-component
+storybook: https://primer.style/view-components/stories/?path=/story/primer-avatar-component
```

Comment:
https://primer.style/view-components/stories is 404ing for me. Do we expect it to work at this point?

Reply:
Yeah, I'm waiting on https://github.com/primer/primer.style/pull/216